.gitignore

```diff
@@ -10,9 +10,6 @@ prontserve-env
 pronterface.spec
 pronsole.spec
 plater.spec
-printrun/gcoder_line.c
-printrun/gcoder_line*.so
-printrun/gcoder_line*.pyd
```

Comment:
Please do not delete as these files will be generated when compiling. This will prevent to upload those files to our repository

Reply:
Fair enough, I'll revert this change.